client/src/components/App.jsx

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
 import axios from 'axios';
+import Reviews from './Reviews.jsx';
 
 export default class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state= {
+      reviews: [],
+      page: 1,
+      sort: 'relevant',
+      product_id: 64620
+    }
+  }
+
+  componentDidMount() {
+    fetch(`/reviews/${this.state.product_id}/${this.state.sort}`)
```

Comment:
This works well on its own and for now. As a team, could we consolidate into one client request somehow? Or will it be best to have a separate client request and get route for each widget?